lib/node_modules/@stdlib/stats/base/dists/normal/ctor/benchmark/benchmark.js

```diff
@@ -270,17 +316,24 @@ bench( pkg+':median', function benchmark( b ) {
 bench( pkg+':mode', function benchmark( b ) {
 	var sigma;
 	var dist;
+	var len;
 	var mu;
+	var x;
 	var y;
 	var i;
 
 	mu = 2.0;
 	sigma = 3.0;
 	dist = new Normal( mu, sigma );
+	len = 100;
+	x = new Float64Array( len );
+	for ( i = 0; i < len; i++ ) {
+		x[ i ] = uniform( EPS + 1.0, 100.0 );
```

Comment:
```suggestion
		x[ i ] = uniform( 1.0 + EPS, 100.0 );
```

Not something that makes a difference, but good for consistency.